week-2/13-order/ignore_this_folder/custom.css

```diff
@@ -0,0 +1,3 @@
+.solution-example {
```

Comment:
I've been putting custom CSS files in a separate CSS directory outside of the exercise directory. https://github.com/NateWr/html-css-git-exercises/tree/master/css

Reply:
Same with the images below. Let's keep them out of the exercises directory: https://github.com/NateWr/html-css-git-exercises/tree/master/images